.github/workflows/ci.yml

```diff
@@ -38,8 +38,6 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
       - name: Install dependencies
-        run: pip install black isort
-      - name: Check with black
-        run: black --check --diff .
-      - name: Check with isort
-        run: isort --check --diff .
+        run: pip install pre-commit
+      - name: Run pre-commit hooks
+        run: pre-commit run --all-files --show-diff-on-failure
```

Comment:
I didn't know about the SKIP environment variable! Yeah, this makes sense. That way if there are mypy failures AND other linter failures, you won't neglect to look at the output here because you see the same mypy warnings from the other steps.